charts/fleet-agent/100.0.3+up0.3.9-rc1/templates/_helpers.tpl

```diff
@@ -0,0 +1,7 @@
+{{- define "system_default_registry" -}}
+{{- if .Values.global.cattle.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
+{{- else -}}
+{{- "" -}}
+{{- end -}}
+{{- end -}}
```

Comment:
we seem to be missing newline to some files

Reply:
I believe this is because they don't exist in source either: https://github.com/rancher/fleet/blob/master/charts/fleet-agent/templates/_helpers.tpl

This would be something each chart owner should address in future PRs